whetstone/runtime-fragment/src/main/java/com/freeletics/mad/whetstone/fragment/ComposeFragment.kt

```diff
@@ -25,45 +25,37 @@ import kotlin.reflect.KClass
  *
  * **Scopes, Dagger and Anvil**
  *
- * There will also be a generated Dagger component that is tied to the composable function but
- * survives configuration changes and stays alive while the composable destination is on the
+ * There will also be a generated Dagger subcomponent that is tied to the Fragment but
+ * survives configuration changes and stays alive while the Fragment destination is on the
  * backstack.
  *
- * The generated component uses [ScopeTo] as it's scope where the [ScopeTo.marker]
- * parameter is the specified [scope] class. This scope can be used to scope classes
- * in the component an tie them to to component's life time. The annotated class is also used as
- * [com.squareup.anvil.annotations.MergeComponent.scope], so it can be used to contribute modules
- * and bindings to the generated component.
+ * The generated component uses [com.freeletics.mad.whetstone.ScopeTo] as it's scope where the
+ * [com.freeletics.mad.whetstone.ScopeTo.marker] parameter is the specified [scope] class. This
+ * scope can be used to scope classes in the component an tie them to to component's life time,
+ * effectively making them survive configuration changes. The  annotated class is also used as
+ * [com.squareup.anvil.annotations.ContributesSubcomponent.scope], so it can be used to contribute
+ * modulesc and bindings to the generated component.
```

Comment:
```suggestion
 * The generated component uses [com.freeletics.mad.whetstone.ScopeTo] as its scope where the
 * [com.freeletics.mad.whetstone.ScopeTo.marker] parameter is the specified [scope] class. This
 * scope can be used to scope classes in the component and tie them to component's life time,
 * effectively making them survive configuration changes. The annotated class is also used as
 * [com.squareup.anvil.annotations.ContributesSubcomponent.scope], so it can be used to contribute
 * modules and bindings to the generated component.
```